manifests/rules/out/imap.pp

```diff
@@ -0,0 +1,7 @@
+# allow outgoing imap
```

Comment:
please add puppet-strings style documentation for the new classes

Reply:
Prefixing with `@summary` would be enough I guess?

Reply:
yes that should do it. afterwards run `bundle exec rake reference`

Reply:
ok, it didn't make a difference